libs/langchain/langchain/llms/edenai.py

```diff
@@ -131,10 +137,12 @@ def _call(
             "providers": self.provider,
             "num_images": 1,  # always limit to 1 (ignored for text)
             "text": prompt,
-            **kwargs,
         }
-        request = Requests(headers=headers)
 
+        if self.model is not None:
```

Comment:
should this just happen in self.params?

Reply:
I'm not sure I understand correctly. Since we moved model to an instantiation parameter as you suggested, we don't use model in self.params anymore.